### DISCUSSION TITLE
pages/UA-PNP/writer_resources/writer_resources_doc/tasks/publishing

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#41] [Q&A]
# pages/UA-PNP/writer_resources/writer_resources_doc/tasks/publishing

You publish content locally and then when your content is ready, you publish to a production server, which for WebHelp is AWS.

https://github01.hclpnp.com/pages/UA-PNP/writer_resources/writer_resources_doc/tasks/publishing.html

**Comment by melanie-feb** [comment]
Test comment
